functions/java-jre.sh

```diff
@@ -17,7 +17,28 @@ java_webupd8() {
   if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 }
 
-java_zulu_embedded() {
+java_zulu() {
+  FILE="$(mktemp /tmp/.XXXXXXXXXX)"
```

Comment:
I don't think there is need to use the temporary file storage here. You make sure to delete the file anyway.

Some opt to make /tmp/ tmpfs, and the archive is > 100MB